admin/src/main/resources/io/buoyant/admin/js/template/usage.template

```diff
@@ -0,0 +1,21 @@
+<h1 class="title">Usage Data</h1>
+
+<p>
+You are seeing this page because you have enabled the <b><i>io.l5d.usage</i></b> telemeter in your <a href="config.json">config file</a>
```

Comment:
TIOLI: We could display on here whether or not `dryRun` is enabled.  That is, whether data is actually being sent to Buoyant or not.

TIOLI, probably out of scope: We could even make `dryRun` mutable and settable from here.  That way, after seeing the payload, users can turn on sending data with a flick of a switch.